mono-traversable/src/Data/MonoTraversable.hs

```diff
@@ -1382,3 +1385,25 @@ ointercalate x = mconcat . List.intersperse x . otoList
         ointercalate x = T.intercalate x . otoList #-}
 {-# RULES "intercalate LText" forall x.
         ointercalate x = TL.intercalate x . otoList #-}
+
+-- | Derives instances of `MonoFoldable`, `MonoFunctor` and `MonoPointed` for types with
+-- instances of `F.Foldable`, `Functor` and `Applicative`.
+--
+-- Useful for, e.g., passing a `F.Foldable` type you don't own into a
+-- function that expects a `MonoFoldable`.
+--
+-- > // package A
+-- > data MyList a = MyList [a] deriving Foldable
+-- >
+-- > // package B
+-- > process :: MonoFoldable mono => mono -> IO ()
+-- >
+-- > // package C
+-- > process (ToMono (MyList []))
+newtype ToMono f a = ToMono { fromMono :: f a }
```

Comment:
Looking at prior art, I see [`WrappedMonad`](https://www.stackage.org/haddock/lts-14.7/base-4.12.0.0/Control-Applicative.html#t:WrappedMonad) in base. Perhaps `newtype WrappedMono f a = WrappedMono { unwrapMono :: f a }`?

Reply:
Agreed. Also changed the wording of the docs to match that of [`WrappedMonoid`](https://hackage.haskell.org/package/base-4.12.0.0/docs/Data-Semigroup.html#t:WrappedMonoid)